### ISSUE TITLE
Can't play with my N64 Hyperkin Adapter anymore.

### ISSUE THREAD

**Issue by TheGhostLibrarian** [Rosalie241/RMG#303]
My N64 Hyperkin Adapter was working fine until the last update. I think it was the SDL controller mode added to the software. Now Rosalie's treating my n64 controller like an Xbox 360 controller. Because of that, the software can't sense the A button, the R-trigger, or even the START button anymore, making it unplayable, which sucks because using an N64 controller is the best way to play these emulators.

**Comment by FuzzyQuils** [comment]
> My N64 Hyperkin Adapter was working fine until the last update. I think it was the SDL controller mode added to the software. Now Rosalie's treating my n64 controller like an Xbox 360 controller. Because of that, the software can't sense the A button, the R-trigger, or even the START button anymore, making it unplayable, which sucks because using an N64 controller is the best way to play these emulators.

Have you tried setting the input plugin to Joystick mode? This can be done via the controller plugin options.

**Comment by TheGhostLibrarian** [comment]
Yeah, but I still couldn't get the START, R-trigger, or B button to work. Like I said, it's treating my N64 controller like a 360 controller.
